weDevsOfficial/dokan
# Feat/Address fields on the vendor registration form

### All Submissions:






### Changes proposed in this Pull Request:


Added the Store Address fields on the vendor registration form with an admin setting.

Closes #.

### How to test the changes in this Pull Request:

1. Enable the `Enable Address Fields` option from the `Dokan > General` setting screen
2. Registration as a vendor from the `My Account` page to see the changes

Screenshots:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/34626059/197401923-f7b66121-7e9e-4595-96f9-41c27a5c17a7.png">
<img width="774" alt="image" src="https://user-images.githubusercontent.com/34626059/197402001-2471c2c7-903d-4f5f-83c2-99b68566c4c6.png">


### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.


### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.


### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.
